tgui/packages/tgui/interfaces/GasTank.js

```diff
@@ -0,0 +1,85 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Box, Section, AnimatedNumber } from '../components';
+import { Window } from '../layouts';
+import { PressureBar } from './common/PressureBar';
+import { ReleaseValve } from './common/ReleaseValve';
+
+export const GasTank = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    pressure,
+    maxPressure,
+    valveIsOpen,
+    releasePressure,
+    maxRelease,
+  } = data;
+
+  const handleSetPressure = releasePressure => {
+    act('set-pressure', {
+      releasePressure,
+    });
+  };
+
+  const handleToggleValve = () => {
+    act('toggle-valve');
+  };
+
+  return (
+    <Window
+      width={400}
+      height={220}>
+      <Window.Content>
+        <Section
+          title="Status">
+          <GasTankInfo
+            pressure={pressure}
+            maxPressure={maxPressure} />
+        </Section>
+        <Section>
+          <ReleaseValve
+            valveIsOpen={valveIsOpen}
+            releasePressure={releasePressure}
+            maxRelease={maxRelease}
+            onToggleValve={handleToggleValve}
+            onSetPressure={handleSetPressure} />
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+
+};
+
+export const GasTankInfo = props => {
+  const {
+    pressure,
+    maxPressure,
+  } = props;
+
+  return (
+    <Fragment>
+      <Box
+        pb="1em">
```

Comment:
Should be able to pass in as `pb={1}`, which should get translated to `1em` behind the scenes but allows for us to change how our paddings/margins are calculated more centrally at a later date.